### DISCUSSION TITLE
optuna terminator - non ML optimization

### DISCUSSION THREAD

**Discussion by okaikov** [optuna/optuna#4719] [Q&A]
hello,
can the newly introduced optuna terminator help us to stop the study if we don't use/have cross validation?
for example I want to optimize a generic problem, what ErrorEvaluator should I use?

**Comment by nzw0301** [comment]
I suppose no; it requires multiple scores.

**Reply by nzw0301** [reply]
I agree with your option, so I've created https://github.com/optuna/optuna/issues/4737.
